cmd/root.go

```diff
@@ -157,30 +170,50 @@ func fprintf(w io.Writer, format string, a ...interface{}) (n int) {
 	return n
 }
 
-// RawFormatter it does nothing with the message just prints it
+// RawFormater it does nothing with the message just prints it
 type RawFormater struct{}
 
 // Format renders a single log entry
 func (f RawFormater) Format(entry *logrus.Entry) ([]byte, error) {
 	return append([]byte(entry.Message), '\n'), nil
 }
 
-func setupLoggers(logFmt string) {
+func setupLoggers(logger *logrus.Logger, logFmt string, logOutput string) error {
```

Comment:
Can we easily move this out of `cmd/` and into the new `log/`?

Reply:
I thought we touch more global variables, but the only ones actually are `logFmt` and `noColor` which are mostly for optimizing so logrus doesn't do stuff loki wouldn't use either way. So I guess ... yes :shrug: , I don't really think it helps ... as `cmd` needs a refactor, which I might try doing once I get a `logger` to each ... package

Reply:
oops `logFmt` is actually not global here, but `stdTTY` is another global used ...